configschema.json

```
@@ -307,6 +307,17 @@
         }
       },
       "required": ["enabled"]
+    },
+    "tiltify": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "enable": { "type": "boolean", "default": true },
+        "clientId": { "type": "string" },
+        "clientSecret": { "type": "string" },
+        "campaign": { "type": "string" }
+      },
+      "required": ["enable", "clientId", "clientSecret", "campaign"]
```

```suggestion
      "required": ["enable"]
```
If set to off by default, the other values need to be made not required otherwise NodeCG will throw a fit if they're not filled either way.